packages/chrome/src/elements/body/Body.tsx

```diff
@@ -11,9 +11,7 @@ import { StyledBody } from '../../styled';
 import { BodyContext } from '../../utils/useBodyContext';
 
 interface IBodyProps {
-  /**
-   * Prepare the body content height to allow space for a footer component
-   **/
+  /** Applies a footer and prepares the body content height for the footer space */
```

Comment:
This prop does not actually add a footer to the `Chrome`. It only "Prepares the body content height for the addition of a footer" (or something to that effect).

Reply:
Updated in latest push.